2022-04-29-history (#1032)
cosname/cosx.org: content/post/2022-04-29-history.md

---
title: 西南联大时期的许宝騄与戴世光
author: 袁卫
date: '2022-04-29'
slug: history-biography-statistician
categories:
  - 新闻动态
tags:
  - 统计史
  - 统计学家

> ---

Reviewer: 这里参照之前文章加上你的名字作为编辑

---
title: 西南联大时期的许宝騄与戴世光
author: 袁卫
date: '2022-04-29'
slug: history-biography-statistician
categories:
  - 新闻动态
tags:
  - 统计史
  - 统计学家

> meta_extra: "编辑：刘姝含"

Reviewer: 此处可以用论坛管理员ID生成一个帖子，然后把帖子URL的编号放过来就可以合并了

![图1 罗常培](https://github.com/SHL12/statictics/blob/main/2.jpg?raw=true) 

![图2 郑天挺](https://github.com/SHL12/statictics/blob/main/3.jpg?raw=true)

### 跑警报

同年2月26日郑天挺日记有:“(下午)一时十分空袭警报作，偕莘田(罗常培)、宝騄贯云大校舍向北山而进，一时二十分紧急警报作，仍前进，一时三十分马街子之紧急警报作，一时秩序殊乱。 军士以敌机至，不准更行。吾辈欲觅一妥善隐蔽之地不可得，复缓移而西，不敢疾步，恐遭干涉也。移至向所常避之某氏坟，席地而坐。一时五十分，闻远处有轰炸声。二时十五分，日重轰炸机二十 七架自西南来，每九架成品字形为一队，三队又作大品字，缓缓向东北而进。机声重浊，高射炮与轰炸机交作，黑烟陡起，似在城中心，敌机未作盘旋，直向东北而去。二时三十分，敌机复自东北折回，阵形如旧，将及城市大炸，黑烟尤甚，似在北门左近。三时十五分，缓步返至新校舍，遇矛尘，谓才盛巷、靛花巷二处必有一炸，余两地均有什物也。四时十五分，解除警报。……余书籍十箱 于二十日移至才盛巷公舍外院东厢楼上，幸亦无恙。”这段日记完整、细致地记录了当时日本飞机空袭轰炸昆明、西南联大师生“跑警报”的景象。1940年和1941年是日本轰炸昆明的高峰时期， 1941 年底陈纳德将军率领的美国“飞虎队”开始和日本空战，到1942年4月共击落了上百架日本飞机，保卫了中国西南后方和滇缅公路补给线。由于“飞虎队”的保护，从1942年起，西南联大师生“跑警报”的情况就少多了。当时的空袭警报分为三种，先是“预行警报”，一般是日本飞机起飞后的预警;接下来第二种是“空袭警报”，表示日本飞机进入云南省了;最后第三种是“紧急警报”，连续短音，表示敌机已经接近昆明。由于许宝騄与郑天挺是邻居，所以郑天挺的日记中有多次记载了他们一起“跑警报”。

### 挽留

1941年3月6日郑天挺写到:“九时江泽涵来，谈算学系事，知许宝騄有至四川大学之意，请泽涵力挽之。人才不易得，既来复放之，大不可也。”许宝騄1月刚到，3月就有调动之意，原因没有提及。但郑天挺和时任算学系主任江泽涵挽留的工作显然是有效的，为西南联大和北大留住了许宝騄。

### 教学与研究

> 从1941年到1945年夏许宝騄在西南联大开过“微积分”、“微分几何”、“近世代数”、“富里哀级数与积分”等课程，特别是首次在我国高校中开设了“数理统计”课程。当时的学生钟开莱(Kai- Lai Zhong，美国斯坦福大学教授)回忆说: “他开了一门很长的课程，从测度和积分开始，讲到概率论，再通向数理统计。课程的第一部分基于Carathéodory Vorlesungenüberreelle Funktionen，第二部分则取材于Cramer的Ｒandom Variables and Probability Distributions。他是一个风度优雅而又富有激情的老师，随身带着预先写在本子上的完整的备课笔记。他喜欢在课堂上点出微妙之处。例如，在推导特征函数的反演公式的时候，他强调Lebesgue-Stieltjes积分是能在单个点上积分的。” (许宝騄先生纪念文集编委会，2010)

Reviewer: ```suggestion
从1941年到1945年夏许宝騄在西南联大开过“微积分”、“微分几何”、“近世代数”、“富里哀级数与积分”等课程，特别是首次在我国高校中开设了“数理统计”课程。当时的学生钟开莱(Kai- Lai Zhong，美国斯坦福大学教授)回忆说: “他开了一门很长的课程，从测度和积分开始，讲到概率论，再通向数理统计。课程的第一部分基于Carathéodory Vorlesungenüberreelle Funktionen，第二部分则取材于Cramer的Ｒandom Variables and Probability Distributions。他是一个风度优雅而又富有激情的老师，随身带着预先写在本子上的完整的备课笔记。他喜欢在课堂上点出微妙之处。例如，在推导特征函数的反演公式的时候，他强调Lebesgue-Stieltjes积分是能在单个点上积分的。” [^1]

```

![图3 西南联大算学系课表](https://github.com/SHL12/statictics/blob/main/4.jpg?raw=true) 

郑天挺1941年4月9日(星期日)记载，“七时起。宝騄隔墙在塌上相告，谓苦思数年一数学 问题已豁然贯通，甚可喜也。”这一段日记今天读来十分生动、逼真、感人。在当时那样艰苦的环境中，居住在相对简陋的单身公寓，白天经常要“跑警报”，许宝騄晚上和邻居郑天挺、罗常培等不同学科的好友谈论国事、校事、学问和生活琐事(郑曰“杂谈”或“上下古今”)，夜深人静时凝思苦想去探索数学和统计问题，当苦思多年没有解决的难题豁然开朗时，兴奋喜悦之情禁不住立即与好友分享，郑天挺虽然是历史学家，但对数学的探索和发现极为理解、尊重，“甚可喜也”是最好的祝贺。当然，我们现在已经无从考证这一难题是哪一个数理统计问题了，但从“苦思数年”的叙述来看一定是许宝騄重要贡献之一了。

1942年，许宝騄由于“数理统计论文”荣获第一届(1941年度) “国家学术奖励金”。该奖励是由当时最高学术审议机构——民国政府教育部学术审议委员会设立的。当年获此殊荣的数学家共两位，另一位是华罗庚。为此，当年设在昆明的“新中国数学会”专门举办茶话会，祝贺华、许二人。

![图4 许宝騄1941年的论文](https://github.com/SHL12/statictics/blob/main/5.jpg?raw=true)

### 通货膨胀

抗战时期昆明物价持续上涨，西南联大的教授们不得不通过各种方式挣些外快，补贴生活。如闻一多先生曾经摆地摊、刻图章养家糊口，而许宝騄和同舍的好友则通过集资得到些收入。郑天挺日记 1942年8月8日写到：“自近三数年物价高涨，校中同人多有营商以资弥补者，余均未加入。近日心恒、宝騄、祥瑞集资万元，欲余加入，言之再四，乃入千五百元。”1943年11月30日继续：“本月共支饭食费一三四三元，酬应费七五五元，零用一三三九元七角，用二0一一元。收入经常四三九八元，临时二0八二元。去年八月，心恒、宝騄集股营商，余入一千五百元。本月退回本金外，盈利九九九元，币值大跌矣。”这两段日记记录了在当时物价飞涨环境下，许宝騄和朋友们集资营商以增加收入的过程。按照日记所写，郑天挺入股1500元，一年三个月后虽盈利999元，但这999元还不够一个月的饭钱或零用钱。当时货币贬值的速度见郑天挺1942年8月4日日记: “晚饭后偕宝騄等至大街，物价较数十日前又二倍馀矣。”正是由于物价几十天就翻番，迫使西南联大教授各显神通，纷纷干起“副业”，许宝騄等即使想尽办法营商，其结果是挣的钱还不如贬值快。

### 唱昆曲

> 西南联大的教授们生活固然艰苦，但他们豁达、乐观，经常聚在一起，文娱体育活动丰富。许宝騄先生在联大教授中因昆曲而出名。余斌《西南联大－昆明记忆》第三册“文化与生活”有这么一篇 “浦江清戏咏 一去二三里 …”[^5]。“西南联大的教授和夫人中，喜好戏曲及传统音乐的不乏其人。1941年夏，老舍应联大中文系邀请来讲学，下榻青云街靛花巷，那里是联大教授宿舍，里面就住着好几位昆曲专家和爱好者。水平最高的要数数理统计学家许宝騄。这位教授是留英博士，中国数理统计学的开创者和奠基人，令人想不到的是，许氏竟会唱三百多出昆曲。另一位是中文系的罗常培教授(老舍的小学同学，满族) ，也学昆曲，目的是要研究制曲与配乐的关系，他是音韵学家。另一位是外文系袁家骅教授的夫人钱晋华女士，她原本就是音乐家。据老舍在《滇行短记》中讲，在昆曲上，许宝騄是后两位的‘老师’。而老舍本人呢，谦虚，说‘我本来学过几句昆曲，到这里也想再学一点。可是，不知怎的一天一天地度过去，天天说拍曲，天天一拍也未拍，只好与许宝騄约定：到抗战胜利后，一同回北平去学，不但学，而且要彩唱 ! ’”虽然老舍与许宝騄聊的是昆曲，但字里行间流露出他们对抗战胜利的乐观态度。不知道的是，回北京后许宝騄和老舍先生是 否实践了他们在昆明靛花巷的约定。

Reviewer: `而且要彩唱 ! `这里多了个空格，可以用 ctrl/cmd + F 找下所有空格，看是不是有问题

### 赴美讲学

许宝騄于1945年6月应美国加洲伯克利大学奈曼教授(J．Neyman)、哥伦比亚大学霍特林(H．Hotelling)邀请赴美讲学。他离开昆明西南联大的情况在郑天挺的日记中也有记载。1945年6月10日有“宝騄来，谈后日即将飞印度转美国矣，不愿使人知，故只告泽涵、毓淮及余三人”。6月12日记“晨大雨，六时闻人声，起送宝騄，则已冒雨行矣”。赴美讲学本应是喜事，但许宝騄当时不愿意声张，只告诉郑天挺、江泽涵(系主任)和程毓淮几位挚友，原因已不能得知，但许宝騄行为做事低调是一贯的作风。

## 3.西南联大时期的戴世光

![从人口革命到建立中国唯物主义统计派](https://github.com/SHL12/statictics/blob/main/6.jpg?raw=true)

### 呈贡文庙

戴世光1938年7月由欧洲取道香港、越南回到昆明。由于在哥伦比亚大学学了一年的经济统 计，又在美国国情普查局和欧洲进修人口普查方法，戴世光受聘清华大学国情普查研究所副教授，并任统计组主任。当时，研究所的所长由西南联大社会学系主任陈达兼任，社会学系李景汉教授也兼任研究所教授、调查组主任。研究所当时设在昆明市近郊呈贡的文庙，文庙是地方政府教育机构祭祀孔子的祠庙建筑，明洪武十六年(1383年)始建于伽宗城(今古城) ，弘治五年(1492年)迁至县城北门外，万历七年(1579年)迁建于呈贡。屡经损毁重修，后仅存崇圣祠、大成殿、东西厢房、棂星门等建筑。其大殿和崇圣祠前场地宽阔，可以在此对普查员(主要是当地小学老师)做大规模培训。研究所迁文庙期间，谢冰心刚到呈贡时曾小住文庙，后来才搬到“默庐”。戴世光为她题写一联赞道: “半间东倒西歪屋，一个千锤百炼人”。

![图5 文庙](https://github.com/SHL12/statictics/blob/main/7.png?raw=true) 

### 人口与农业普查

> 关于国情普查研究所的工作，陈达在《浪迹十年之联大琐记》有较细致的记载(陈达，2013)［6］。“民国二十七年(1938年)，余于离蒙自① 前一个月，接清华聘书一件，聘余为国情普查研究所所长，学校决定以预算的一部设立五研究所，内四所与自然科学有关:即:(一)属，(二)无线电，(三)航空，(四)农业，其第五所与社会科学有关，特别注重人口，经费每年四万元，七月末旬戴世光君自欧洲归国，已由学校聘为本所职员，到蒙自时关于研究有所商谈。余拟选一试验区，作小规模的人口普查，请戴君协助。戴君前在清华经济系毕业，考得游美公费，专修统计学，注重人口统计。同时请李景汉兄指导，特别关于调查方面。

Reviewer: ```suggestion
关于国情普查研究所的工作，陈达在《浪迹十年之联大琐记》有较细致的记载［^6］。“民国二十七年(1938年)，余于离蒙自① 前一个月，接清华聘书一件，聘余为国情普查研究所所长，学校决定以预算的一部设立五研究所，内四所与自然科学有关:即:(一)属，(二)无线电，(三)航空，(四)农业，其第五所与社会科学有关，特别注重人口，经费每年四万元，七月末旬戴世光君自欧洲归国，已由学校聘为本所职员，到蒙自时关于研究有所商谈。余拟选一试验区，作小规模的人口普查，请戴君协助。戴君前在清华经济系毕业，考得游美公费，专修统计学，注重人口统计。同时请李景汉兄指导，特别关于调查方面。

```

余于(1938年)十二月二日由上海至昆明时，研究所同仁到站相迎，有戴世光、倪因心、李作猷、沈如瑜诸君，研究所在青云街169号，临翠湖，颇幽静。

戴世光君对于选择试验区事，已着手进行，除路远者不适宜外，余亲赴昆明市近外视察，先到昆明县属官渡镇及其近村。官渡离昆明市南门外约五里，一日下午与戴步行去，归时沿滇越铁路走， 遇火车来到九门里时，车虽徐行但不停，余等一跃登车。

余与戴君另一日往呈贡见县长，并到回龙乡在乡公所与乡长、保甲长、小学教员相谈。余有意选呈贡县为试验区。

二月五日(1939年)梅校长与余在研究所设宴，请新任民政厅长李培天(子厚)先生、呈贡县长李晋笏(右侯)先生，及本所同仁，商议人口普查进行事宜。十二日在呈贡开成立会，到会委员有李县长、县党部邓迪民书记长、李景汉、戴世光两兄，此外尚有梅校长、叶企孙先生、教育厅代表、地方各团体代表及本所同仁[^6]。

陈达先生回忆录清楚地记录了国情普查研究所成立，以及陈达、戴世光选定呈贡县作为试验区的过程。

1940年“呈贡人口普查”和1941年“呈贡农业普查”被认为是中国现代国情普查的开始，普查工作结束后，陈达、李景汉、戴世光主编完成了《云南呈贡县人口普查报告》、《云南呈贡农业普查报 告》，继而他们又撰写了《昆明市环湖户籍示范》。由于戴世光在国情普查所工作出色，1940年被晋升为研究所教授。

正是基于呈贡人口普查的经验，戴世光在建国后实施的全国人口普查中提供了很好的建议。在1982年全国第三次人口普查前的无锡试点工作会议上，戴世光提出普查的三个要点:第一是“画地图”;第二是“抓房子”;第三是“查常住”。我国几次人口普查的实践证明，戴世光提出的三个要点反映了搞好人口普查的基本规律，其基本原则到现在还适用。

“昆明环湖户籍示范”调查与研究，除了研究所教授们的辛勤工作，西南联大校方也积极联系当地政府。《梅贻琦西南联大日记》收录了梅贻琦1941年1月1日至1946年10月19日在西南联大期间的日记。由于梅贻琦校长公事繁忙，每晚只是三言两语简要记载当日活动标题，详细内容很少提及。梅校长日记只有21万字，远不及郑天挺日记详细，但其中约有十余处提到戴世光。1942年11月12日就谈到“户籍示范”的工作:“下午五点约陈善初及户籍示范委会诸君会谈，到者为陈(善初)及王子祜、孟立人、高直青、傅科长(警务处) 、陈、李、戴、姚，并留晚饭。

### 授课

戴世光1938年回国后虽然正式工作是在国情普查研究所，但他从1938年秋季学期开始就给法商学院经济系上统计课，陆续开过“统计学”、“初等统计学”、“高等统计学”、“人口统计”和“经济统计”等课程。

> 清华国情普查研究所设在呈贡县的文庙，呈贡位于昆明市东南郊区，距昆明市区约 30 公里。当时住在文庙的戴世光要到城里给学生上课，经常和吴泽霖等教授一起徒步十里到呈贡火车站，然后搭 乘滇越铁路火车到昆明，住在西南联大教师宿舍集中上两天课再返回呈贡。中国人民大学教授、西南联大毕业生林富德回忆戴世光时说: “我和戴老师有很多年接触。1938 年他回来后在清华教书，当时他住在昆明郊区，除了上课以外不大进城。在昆明时我没有上过他的课，但我知道有这么一位年轻老师。怎么知道的呢? 戴老师上课很开放，他在介绍自己姓名时，自称 “Put the world’s light on my head”，这样学生都很接受他。当时的课堂很注意课堂气氛，启发学生。”(戴世光，2008)

Reviewer: ```suggestion
清华国情普查研究所设在呈贡县的文庙，呈贡位于昆明市东南郊区，距昆明市区约 30 公里。当时住在文庙的戴世光要到城里给学生上课，经常和吴泽霖等教授一起徒步十里到呈贡火车站，然后搭 乘滇越铁路火车到昆明，住在西南联大教师宿舍集中上两天课再返回呈贡。中国人民大学教授、西南联大毕业生林富德回忆戴世光时说: “我和戴老师有很多年接触。1938 年他回来后在清华教书，当时他住在昆明郊区，除了上课以外不大进城。在昆明时我没有上过他的课，但我知道有这么一位年轻老师。怎么知道的呢? 戴老师上课很开放，他在介绍自己姓名时，自称 “Put the world’s light on my head”，这样学生都很接受他。当时的课堂很注意课堂气氛，启发学生。”[^2]

```

---
title: 西南联大时期的许宝騄与戴世光
author: 袁卫
date: '2022-04-29'

> slug: history

Reviewer: 这个slug其实可以指向性强一些

categories:
  - 新闻动态
tags:
  - 统计史
  - 统计学家
---

> #西南联大时期的许宝騄与戴世光

Reviewer: `#` 后要有空格，否则不被解析为标题，通篇需要修改

##1.引言

> 许宝騄(1910－1970)与戴世光(1908－1999)是我国近现代统计学界的两面旗帜。许宝騄是中国数学家、统计学家，北京大学一级教授。1928年入燕京大学学习化学，1930年转入清华大学攻读数学，毕业后在北京大学数学系任助教。1936年赴英留学，在伦敦大学、剑桥大学学习，1938年获哲学博士学位，1940年又获科学博士学位。同年回国，任北京大学教授，执教于昆明西南联合大学。1945年再次出国，应邀先后在美国伯克利加州大学、哥伦比亚大学和北卡罗来纳大学任访问教授。1947年10月回国，此后一直在北京大学任教授。1948年当选中央研究院院士。1955年当选中国科学院学部委员。他在多元统计分析与统计推断方面发表了一系列出色的论文，成为我国在数理统计学和概率论研究领域达到世界先进水平的一位杰出学者和一代统计宗师(许宝騄先生 纪念文集编委会，2010)。

Reviewer: 参考文献可以用脚注形式，可以参考[这里](https://bookdown.org/yihui/blogdown/output-format.html)，用`...统计宗师[^1]`，然后参考文献那边可以写成 `[^1]: 许宝騄先生纪念文集编委会．道德文章垂范人间，纪念许宝騄先生百年诞辰［M］．北京: 北京大学出版社，2010. ` 当然如果文献过多，可以考虑用bib文件。

戴世光是我国经济统计学家、人口学家。1927年入清华大学经济学系学习，1931年毕业进入清华大学研究院，1935年赴美留学，1936年获密歇根大学统计学硕士学位，随后入哥伦比亚大学学习经济统计。1937年到美国国情普查局实习并到英国、德国和印度等国考察人口统计。1938年回国在西南联合大学清华国情普查研究所和西南联大经济系任教，先后任副教授、教授。1952年由于院系调整由清华大学转入中央财经学院，1953年转入中国人民大学，1981年被国务院学位委员会批准为首批博士生导师。他提出了“国际科学界只存在一门统计学，它是各国广泛应用的，也是我国对自然科学和社会、经济科学进行研究的一个必要的科学方法、技术”，推动了我国统计科学与教育的现代化(戴世光，2008)。

抗日战争时期，国立北京大学、国立清华大学和私立南开大学内迁到昆明，在1938年4月设立国立西南联合大学(西南联大)。当时，西南联大下设文、理、工、法商和师范5个学院，讲授统计课程的共有三位老师，分别是理学院算学系(数学系)的许宝騄，法商学院经济学系的杨西孟① 和戴世光。许宝騄在算学系首次开设了“数理统计”课程，杨西孟和戴世光则在法商学院开设“统计学”系列课程。许宝騄和戴世光的文集，以及回忆、纪念文章中关于二位先生在西南联大的教学、研究和日常生活的记载较少，学界后人一直渴望了解他们在西南联大艰苦的岁月里是怎样教书育人、怎样作研究，以及又是怎样乐观地生活。近期笔者在搜集整理统计学史料时看到两本新近出版的日记———《梅贻琦西南联大日记》(梅贻琦，2018)和《郑天挺西南联大日记》(郑天挺，2018)，继而进一步查阅西南联大相关的文献，大致勾画出了戴世光、许宝騄在西南联大的工作和生活。

##2.西南联大时期的许宝騄

> !(https://github.com/SHL12/statictics/blob/main/1.jpg?raw=true) (中国概率论与数理统计研究的开拓者之一)

Reviewer: 链接格式不对 应使用这样的格式 `![图片说明](链接)`

1946年2月1日除夕，“阴冷。昨夜有风，天气又冷矣。一日家人甚忙，备晚间年饭各事。六点与泽霖赴土司官刀多诸君饭约于翠湖招待所，廊前设席，风冷且不整齐，灯光又暗。印堂、勉仲之外，更无他客。盖除夕人家多不出门矣。八点余未待终席辞出。至家客人大小廿七八人(查、章、姚、毕四家及愫斐瑏瑠、松龄、祖培、钟安度瑏瑡、蒋燕华瑏瑢、陶燕锦瑏瑣、全绍志瑏瑤、戴世光) 已坐立堂屋待进晚餐，挤挤(济济) 一堂极为热闹。饭后分作冠圈、牌九等戏，三点食元宵年糕，又闲谈至五点，客大部散去。收拾屋中零事，至六点半天已明，始上床睡”。除夕日，梅校长先是在下午6点与当地少数民族头领礼节性聚会，晚8点多回家与家人和亲朋好友一起过年。在家里请来的客人中，多数是梅家的老朋友，如北大蒋梦麟校长的两位女儿，跟随自己的秘书、医生等，教授出席的很少。但戴世光被邀赴梅校长家宴，也可看出梅校长通过译员训练班对当时年轻的戴世光(38岁)很是赏识，北归清华园后让戴世光出任清华一年级主任。年夜饭后大家打牌聊天，又吃元宵年糕，一直热闹到早上5点才离去，迎接抗战胜利后的新春。

###篮球赛

戴世光身高1米8多，1927－1931年在清华是学校篮球队的主力之一，而在进入清华之前，戴世光的高中是在南开中学读的，也曾是南开中学篮球队队员。1939年元旦，刚成立不久的西南联大为了庆祝新年，举办了三校的篮球赛。在清华与南开两校的对阵中，31岁的戴世光上场了，此时清华大学队比分已经落后，清华学生拉拉队摇动校旗，高喊: “不要慌，不要忙，清华来了戴世光。” 戴世光身穿清华球衣，带着眼镜上场，但毕竟好几年没怎么打球，加上体力也不如年轻人，打了一会儿就被年轻队员换下了。最后，南开篮球队战胜了清华和北大，获得冠军。

许宝騄和戴世光两位先生虽然都离我们而去了，但他们当年播下的统计人才培养、科学研究和统计教育改革创新的种子已经硕果累累，他们爱国勤奋的精神和无私奉献的情操将永远激励我们向前。

##参考文献

［1］许宝騄先生纪念文集编委会．道德文章垂范人间，纪念许宝騄先生百年诞辰［M］．北京: 北京大学出版社，2010．

［2］戴世光．戴世光文集［M］．北京: 中国人民大学出版社，2008．

［3］梅贻琦．梅贻琦西南联大日记［M］．北京: 中华书局，2018．

［4］郑天挺．郑天挺西南联大日记［M］．北京: 中华书局，2018．

［5］余斌．西南联大 ·昆明记忆［M］．昆明: 云南民族出版社，2003．

［6］陈达．浪迹十年之联大琐记［M］．北京: 商务印书馆，2013．

［7］赵培中．吴泽霖执教 60 周年暨 90 寿辰纪念文集［M］．武汉: 湖北科学技术出版社，1988．

［8］西南联合大学史料［M］．昆明: 云南教育出版社，1998．

> ##作者简介

Reviewer: 主站已经有袁老师个人信息，所以应删掉这部分
